packages/server/tests/clients/servicesClient.ts

```diff
@@ -286,6 +287,14 @@ export default class ServicesClient {
         return accountBalance.hbars;
     }
 
+    async getFileContent(fileId: string): Promise<any> {
+        const query = new FileContentsQuery()
+            .setFileId(fileId);
+
+        const data = await query.execute(this.client);
```

Comment:
nit: we can skip the variable assignment `return await query.execute(this.client)`